src/tcgwars/logic/impl/gen3/LegendMaker.groovy

```diff
@@ -1611,12 +1611,12 @@ public enum LegendMaker implements LogicCardInfo {
           onAttack {
             def pcs = defending
             if (opp.bench) {
-              if (confirm("Switch 1 of your opponent’s Benched Pokémon with the Defending Pokémon.")) {
-                pcs = opp.bench.oppSelect()
+              if (confirm("Switch 1 of your opponent’s Benched Pokémon with the Defending Pokémon?")) {
```

Comment:
The text implies this should not be a choice.

Reply:
Fixed

Reply:
No, I mean you shouldn't use a `confirm` here. The text should go on the `select` instead, and the confirm should be removed.

Reply:
Oh there was more wrong with this card than I thought haha. That should do it.